File: desktop-qt-widgets/question.cpp
#include "question.h"

Question::Question(QObject *parent) : QObject(parent)
{

}

QString Question::getQuestion() const
{
    return question;
}

void Question::setQuestion(const QString &value)
{
    question = value;
}

QStringList Question::getAnswers() const
{
    return answers;
}

void Question::setAnswers(const QStringList &value)
{
    answers = value;
}

int Question::getCorrectAnswerIndex() const
{
    return correctAnswerIndex;
}

void Question::setCorrectAnswerIndex(int value)
{
    correctAnswerIndex = value;
}

File: desktop-javafx/src/main/java/com/fattysmurff/fiddlefaddle/javafx/Question.java
package com.fattysmurff.fiddlefaddle.javafx;

import java.util.ArrayList;
import java.util.List;

public class Question {

    private int correctAnswerIndex = 0;
    private String Question = "";
    private List<String> answers = new ArrayList<>();

    public int getCorrectAnswerIndex() {
        return correctAnswerIndex;
    }

    public void setCorrectAnswerIndex(int correctAnswerIndex) {
        this.correctAnswerIndex = correctAnswerIndex;
    }

    public String getQuestion() {
        return Question;
    }

    public void setQuestion(String question) {
        Question = question;
    }

    public List<String> getAnswers() {
        return answers;
    }

    public void setAnswers(List<String> answers) {
        this.answers = answers;
    }
}
